storage_service/storage_service/settings/local.py

```diff
@@ -61,6 +61,11 @@
 # )
 # ########## END TOOLBAR CONFIGURATION
 
+# ######## AUTHENTICATION CONFIGURATION
+# Disable password validation in local development environment.
+AUTH_PASSWORD_VALIDATORS = []
```

Comment:
This was helpful for testing! Thanks for thinking of it.

Reply:
Glad you liked it! I wrestled with whether this was a good idea, but I think ultimately it's more useful than not.